### ISSUE TITLE
Protein sequence abundance

### ISSUE THREAD

**Issue by cmorganl** [soedinglab/plass#13]
Hi there,

I've just started using PLASS, specifically `plass assemble`, and I really like it!

This isn't an issue but a potential enhancement. I was wondering how I might be able to recover abundance information for each protein assembled. Its not as simple as with nucleotide contigs since I'm unable to align the reads back to the assembly in this case.

Could the abundance of proteins be included in the header, as SPAdes does? Alternatively, a table mapping header to its respective abundance would be convenient as well. I hope I haven't misinterpreted the output and its already provided :)

Thanks!
Connor

**Comment by martin-steinegger** [comment]
Yes I would love to have this as well. 👍 
But so far I did not come up with an solution how I could gather this information.

**Comment by apcamargo** [comment]
Wouldn't it possible to use an EM algorithm to "distribute" the read among its multiple hits?

**Comment by martin-steinegger** [comment]
@apcamargo yes this a good idea. But do you know if EM would be fast enough to handle this large amount of data?

**Comment by apcamargo** [comment]
I don't know. It would depend on the total amount of proteins and the number of orthologs. I imagine it would be much slower than a typical RNA-Seq quantification, but not unfeasible. But that's just my impression, I've never done something like that.

**Comment by mooreryan** [comment]
I was just thinking about this as well...are there any current plans to include this in plass, or would it be better for now to try the `mmseqs map` workflow?
